### ISSUE TITLE
Bundle all JS artifacts

### ISSUE THREAD

**Issue by hoerup** [salesforce/sloop#208]
Currenly a handfull of js artifact ar fetched from 3rd party CDN's

Consider bundling these for use in eg airgapped environments

**Comment by hongruiji** [comment]
Agree, currently cannot run offline, need to connect to the network。
